homeassistant/components/rfxtrx/__init__.py

```diff
@@ -191,7 +190,14 @@ async def async_setup_entry(hass, entry: config_entries.ConfigEntry):
 
     hass.data[DOMAIN][DATA_CLEANUP_CALLBACKS] = []
 
-    await async_setup_internal(hass, entry)
+    try:
+        await async_setup_internal(hass, entry)
+    except asyncio.TimeoutError:
+        # Library currently doesn't support reload
+        _LOGGER.exception(
```

Comment:
```suggestion
        _LOGGER.error(
```

We should be printing backtrace here. We know the cause.